src/Components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,49 @@
+import "./navbar.css";
+import { Link } from "react-router-dom";
+
+
+export  function Navbar  () {
+    return <nav className="ecom-navbar">
+        <div class="flex-center">
+            <ion-icon class="ham-menu" name="menu-outline"></ion-icon>
+            <div>
+                <Link to="/HomePage">
```

Comment:
use `/` route because it is defined in your routes and mostly used for Homepage.